packages/core/src/types/index.ts

```diff
@@ -1,10 +1,19 @@
-import { Camera, Frustum } from '@babylonjs/core';
-import { Tile, PriorityQueue } from '@tiledb-inc/viz-common';
+import { BoundingInfo } from '@babylonjs/core';
+import { Matrix } from 'mathjs';
+
+export type TDBNonEmptyDomain = {
+  isEmpty: boolean;
+  nonEmptyDomain: Record<string, number[]>;
+};
 
 export interface AssetMetadata {
-  dataset_type: string;
+  dataset_type?: string;
```

Comment:
can you add a comment about what values of `dataset_type` are valid?

does this really want to be an enum?